roles/postgres-cluster-pgbouncer-client-setup/templates/userlist.txt.j2

```diff
@@ -1,4 +1,4 @@
 "{{ postgresql_cluster_pg_bouncer_root_user | default('pgbounceradmin') }}" "{{ postgresql_cluster_pg_bouncer_root_password_hashed | default('md5' + ((postgresql_cluster_pg_bouncer_root_password + (postgresql_cluster_pg_bouncer_root_user | default('pgbounceradmin'))) | hash('md5'))) }}"
```

Comment:
this is not using scram-sha-256

Reply:
I have never used pg_bouncer before. Can I just login to the 6432 port with the pgbounceradmin user? I am not 100% sure what this account is for, but I can definitely do some more research and figure it out as well. 

How is this user getting added to postgres as I don't see them being added anywhere. Unless I am missing something.  For me to get the correct scram-sha-256 hashed password I have to pull it directly from Postgres.

I wil DMOR tonight and see what I can figure out here. Everything was working, so I didn't understand fully what this account was used for to be fair.